scripts/authz_proposal_for_validator_ops.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -e
+
+# Instructions:
+#  - to run: ./authz_proposal_for_validator_ops.sh [grant/revoke]
+#  - put grantee addresses in grantee.txt
+
+if [[ -z "$1" ]]; then
+    operation="grant"
+else
+    operation=$1 # should be "revoke"
+fi
+
+rm /tmp/grant-*.json || true
+rm /tmp/revoke-*.json || true
+
+if [[ -z "${GRANTER}" ]]; then
+    echo -n GRANTER:
+    read granter 
+    export GRANTER="${granter}"
+    echo
+    printf 'Please run "export GRANTER=%s" to avoid setting it in the future or add it to your bashrc file.'
+    echo 
+    echo
+else
+    echo "\$GRANTER is set to '$GRANTER'"
+fi
+
+if [[ -z "${CHAIN_ID}" ]]; then
+    echo -n CHAIN_ID:
+    read chain_id 
+    export CHAIN_ID="${chain_id}"
+    echo
+    printf 'Please run "export CHAIN_ID=%s" to avoid setting it in the future or add it to your bashrc file.'
+    echo 
+    echo
+else
+    echo "\$CHAIN_ID is set to '$CHAIN_ID'"
+fi
+
+printf 'Grantees (reading from grantee.txt):'
+echo
+for GRANTEE in $(cat grantee.txt); do
+    echo " - $GRANTEE"
+done
+
+if [[ $operation == "enable" ]]
```

Comment:
above you're using "grant/revoke" but here you check for "enable"

Reply:
updated